templates/csv.yaml

```diff
@@ -467,8 +467,10 @@ spec:
                       value: "deploy/resources/quickstart/serverless-application-quickstart.yaml"
                     - name: DASHBOARDS_ROOT_MANIFEST_PATH
                       value: "deploy/resources/dashboards"
-      
-      # Openshift Ingress adapter for Knative's Ingress implementation.
```

Comment:
Please don't remove this.

Reply:
ok probably removed by accident will bring it back.